crates/router/src/connector/multisafepay/transformers.rs

```diff
@@ -64,6 +65,8 @@ pub enum Gateway {
     Klarna,
     Googlepay,
     Paypal,
+    Ideal,
+    Giropay,
```

Comment:
Please add Currency and Country configs for both Ideal and Giropay in `config/development.toml`, `config/deployments/sandbox.toml`, `config/deployments/integration_test.toml`, `config/deployments/production.toml` for Multisafepay